dephell/config/scheme.py

```diff
@@ -20,6 +20,7 @@
 
 # all fields with default value (defaults.py) marked as required.
 SCHEME = {
+    'owner': dict(type='string', required=True),
```

Comment:
It's not required :) `required=False`

Reply:
`SCHEME` content grouped by sections. You will place this option in `build_other`. So, place it after `# other` here, please.

Reply:
Changed this to False.

Reply:
Oh okay, I'll change the placement.